docs/source/recipes/i18n.md

```diff
@@ -160,7 +163,42 @@ As the output suggests it will first extract all messages from the source files
 Then it will synchronize the extracted messages with the `.pot` master template and with all the `.po` files found in the project.
 This script will combine the messages located in Volto itself and the current project, and combine them into the `.json` files.
 
-## Contributing translations for a unsupported language
+
+## Overriding i18n messages
+
+If you want to override an existing translation, you should declare the original message
+again somewhere else in your project, e.g in `src/config.js`, like this:
+
+```js
+import { defineMessages } from 'react-intl';
+
+defineMessages({
+  back: {
+    id: 'Back',
+    defaultMessage: 'Back',
+  },
+});
+```
+
+Then run `yarn i18n` and you'll find the translation ready to override in your `locales`
+directory, e.g `locales/de/LC_MESSAGES/volto.po`.
+
+```
+#: src/config
+msgid "Back"
+msgstr "My overriden translation"
+```
+
+After setting the override, then run `yarn i18n` again, to create the `de.json`
+translation files. Restart Volto to see the changes applied.
+
+!!! note
+    Shadowed components do NOT override translations, since 99% of the time, you
+    don't want them to do that, so the `customizations` folder is excluded from the
+    i18n build.
+
```

Comment:
Make above block MyST-y.

````
```{note}
Shadowed components do _not_ override translations.
99% of the time you do not want them to do that.
Thus the `customizations` folder is excluded from the i18n build.
```
````